docs/product-updates/2023-06-07-api-branch-management.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Simplify Your API Workflow with Bump.sh's API New Branch Management Feature
+tags: [New feature]
```

Comment:
:tipping_hand_person: suggestion: I would mention the API as a general tag for this update as it's only changing the Bump API.

```suggestion
tags: [New feature, Bump API]
```

What do you think?